### PR TITLE
EN-2268 sorting tabs on top

### PR DESCRIPTION
**Pull request by gidesan**
I'm writing tests, so feel free to suggest improvements but please don't merge.

I added some state for filters, it can be a raw starting point for https://jira.entando.org/browse/EN-2325

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/digital-exchange/CategoryTabsContainer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+};
+
+export const mapDispatchToProps = dispatch => ({
+  onWillMount: () => (dispatch(fetchDECategories())),
+  onSelect: (category) => {
+    const filters = {
+      formValues: { category: category ? [category] : [] },
+      operators: FIELD_OPERATORS,
+    };
+    // TODO proposal for https://jira.entando.org/browse/EN-2325: pass filters to fetchDEComponents and dispatch set filters action there
```
Comment (nicpuddu) [inline src/ui/digital-exchange/CategoryTabsContainer.js]: lets avoid TODO comments in code unless we know for a fact that they are gonna be gone within a few PRs

File: src/ui/digital-exchange/CategoryTabs.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    return (
+      <div className="CategoryTabs">
+        <Tabs
+          id="de-category-tabs"
+          justified
+          defaultActiveKey={0}
+          onSelect={key => handleSelect(key)}
+        >
+          {
+            categories.map((category, index) => (
```
Comment (nicpuddu) [inline src/ui/digital-exchange/CategoryTabs.js]: for a matter of cleaningness lets avoid array maps within the body of the return. put this on a const before the return statement

**Comment by entando-service** [comment]
All is well

**Comment by gidesan** [comment]
tests added, so it can merged when it passes the code review

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**